### ISSUE TITLE
+datascript option

### ISSUE THREAD

**Issue by pasviegas** [martinklepsch/tenzing#31]
Well, datascript is a must have for any cljs app that aims to be offline, and also really good for prototyping.

Also, this is a very good addtion if we are expecting om.next to be used soon :)

**Comment by martinklepsch** [comment]
I agree that datascript is nice but I'd like to keep most of the options to this template to add something to the build setup. Them Om and Reagent options make an exception to that but those seem to be very, very common ground.
